frontend/app/src/composables/use-pagination-filter/index.ts

```diff
@@ -33,7 +33,7 @@ interface UsePaginationFiltersOptions<
   locationOverview?: Ref<string>;
   filterSchema?: () => FilterSchema<TFilter, TSuggestionMatcher>;
   onUpdateFilters?: (query: LocationQuery) => void;
-  extraParams?: ComputedRef<RawLocationQuery>;
+  extraParams?: ComputedRef<RawLocationQuery> | Ref<RawLocationQuery>;
```

Comment:
Any reason for this change?

Reply:
I wanted to wrap the extraParams inside refDebounced, so if user click `next month` too fast, it's efficient.
But nevermind, we can remove it, I can refDebounced the dependency instead